### PR TITLE
Feature multinode

### PR DESCRIPTION
**Pull request by al1img**

### CODE REVIEW

**Review by Eabramov84 — APPROVED** [review]

File: meta-aos-rcar-gen3/meta-aos-rcar-gen3-dom0/conf/layer.conf [hunk 1]
```diff
@@ -7,6 +7,6 @@ BBFILES += "${LAYERDIR}/recipes-*/*/*.bb \
 
 BBFILE_COLLECTIONS += "aos-rcar-gen3-dom0"
 BBFILE_PATTERN_aos-rcar-gen3-dom0 := "^${LAYERDIR}/"
-BBFILE_PRIORITY_aos-rcar-gen3-dom0 = "7"
+BBFILE_PRIORITY_aos-rcar-gen3-dom0 = "14"
```
Comment (oleksiimoisieiev) [inline meta-aos-rcar-gen3/meta-aos-rcar-gen3-dom0/conf/layer.conf:10]: Why did you change priority from 7 to 12 and then from 12 to 14? I thought we've discussed around 12

File: meta-aos-rcar-common/meta-aos-rcar-common-dom0/recipes-guests/domd/files/domd-set-root [hunk 2]
```diff
@@ -7,16 +7,40 @@ DOMD_CFG_FILE="/etc/xen/domd.cfg"
 BOOT_STORAGE=$(cat /proc/device-tree/boot_dev/device)
 
 if [ -z "$BOOT_STORAGE" ]; then
-    BOOT_STORAGE=mmcblk1
-
     echo "WARNING! Using default storage: ${BOOT_STORAGE}"
+
+    BOOT_STORAGE=mmcblk1
```
Comment (oleksiimoisieiev) [inline meta-aos-rcar-common/meta-aos-rcar-common-dom0/recipes-guests/domd/files/domd-set-root:12]: Why?

**Comment by oleksiimoisieiev** [comment]
Looks good apart from minor comments
